Build with Hedgehog 1.2

Unfortunately just bumping the bounds isn't enough:

```
[ 1 of 44] Compiling Hedgehog.Classes.Common.Bottom ( src/Hedgehog/Classes/Common/Bottom.hs, dist/build/Hedgehog/Classes/Common/Bottom.o, dist/build/Hedgehog/Classes/Common/Bottom.dyn_o )
[ 2 of 44] Compiling Hedgehog.Classes.Common.Compat ( src/Hedgehog/Classes/Common/Compat.hs, dist/build/Hedgehog/Classes/Common/Compat.o, dist/build/Hedgehog/Classes/Common/Compat.dyn_o )
[ 3 of 44] Compiling Hedgehog.Classes.Common.Func ( src/Hedgehog/Classes/Common/Func.hs, dist/build/Hedgehog/Classes/Common/Func.o, dist/build/Hedgehog/Classes/Common/Func.dyn_o )
[ 4 of 44] Compiling Hedgehog.Classes.Common.ApTrans ( src/Hedgehog/Classes/Common/ApTrans.hs, dist/build/Hedgehog/Classes/Common/ApTrans.o, dist/build/Hedgehog/Classes/Common/ApTrans.dyn_o )
[ 5 of 44] Compiling Hedgehog.Classes.Common.Gen ( src/Hedgehog/Classes/Common/Gen.hs, dist/build/Hedgehog/Classes/Common/Gen.o, dist/build/Hedgehog/Classes/Common/Gen.dyn_o )
[ 6 of 44] Compiling Hedgehog.Classes.Common.Equation ( src/Hedgehog/Classes/Common/Equation.hs, dist/build/Hedgehog/Classes/Common/Equation.o, dist/build/Hedgehog/Classes/Common/Equation.dyn_o )
[ 7 of 44] Compiling Hedgehog.Classes.Common.IO ( src/Hedgehog/Classes/Common/IO.hs, dist/build/Hedgehog/Classes/Common/IO.o, dist/build/Hedgehog/Classes/Common/IO.dyn_o )
[ 8 of 44] Compiling Hedgehog.Classes.Common.PP ( src/Hedgehog/Classes/Common/PP.hs, dist/build/Hedgehog/Classes/Common/PP.o, dist/build/Hedgehog/Classes/Common/PP.dyn_o )

src/Hedgehog/Classes/Common/PP.hs:38:13: error:
    • The constructor ‘Report’ should have 5 arguments, but has been given 4
    • In the pattern: Report tests discards coverage status
      In an equation for ‘ppResult’:
          ppResult r@(Report tests discards coverage status)
            = case status of
                Failed (FailureReport size seed shrinks _mcoverage annots _mspan
                                      msg _mdiff footnotes)
                  -> let ...
                     in R.ppResult Nothing (Report tests discards coverage failure)
                _ -> R.ppResult Nothing r
   |
38 | ppResult r@(Report tests discards coverage status) = case status of
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

--- a/hedgehog-classes.cabal
+++ b/hedgehog-classes.cabal
@@ -151,7 +151,7 @@ library
     , base >= 4.12 && < 4.18
     , binary >= 0.8 && < 0.9
     , containers >= 0.5 && < 0.7
-    , hedgehog >= 1 && < 1.2
+    , hedgehog >= 1.2 && < 1.3
     , pretty-show >= 1.9 && < 1.11
     , silently >= 1.2 && < 1.3
     , transformers >= 0.5 && < 0.6

--- a/src/Hedgehog/Classes/Common/PP.hs
+++ b/src/Hedgehog/Classes/Common/PP.hs
@@ -35,8 +35,8 @@ renderResult x = renderDoc u =<< ppResult x
 ppResult :: MonadIO m
   => Report Result
   -> m (Doc Markup)
-ppResult r@(Report tests discards coverage status) = case status of
-  Failed (FailureReport size seed shrinks _mcoverage annots _mspan msg _mdiff footnotes) ->
-    let failure = Failed $ FailureReport size seed shrinks Nothing annots Nothing msg Nothing footnotes
-    in R.ppResult Nothing (Report tests discards coverage failure)
+ppResult r@(Report tests discards coverage seed status) = case status of
+  Failed (FailureReport shrinks shrinkPath _mcoverage annots _mspan msg _mdiff footnotes) ->
+    let failure = Failed $ FailureReport shrinks shrinkPath Nothing annots Nothing msg Nothing footnotes
+    in R.ppResult Nothing (Report tests discards coverage seed failure)
   _ -> R.ppResult Nothing r

